animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -276,16 +286,19 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
         iSin = sin(startAngle)
         stepCos = cos(angleStep)
         stepSin = sin(angleStep)
+        
+        Zoffset = altitude / 2 * int(self.centerAxis)
```

Comment:
We don't use capital letters as the start of variable names. Also, put parentheses for clarity here:
```
zOffset = (height / 2.0) * self.centerSpiral
```